Description: Several bugs fixed for the free manpage:
 -g flag documented #286900
 hypen escaped #282168
 Reformated completely #534198
 free -o better documented #526604
Bug-Debian: http://bugs.debian.org/286900
Bug-Debian: http://bugs.debian.org/282168
Bug-Debian: http://bugs.debian.org/534198
Bug-Debian: http://bugs.debian.org/526604
Author: <csmall@debian.org>
--- a/free.1
+++ b/free.1
@@ -1,47 +1,67 @@
 .\"             -*-Nroff-*-
 .\"  This page Copyright (C) 1993 Matt Welsh, mdw@sunsite.unc.edu.
 .\"  Freely distributable under the terms of the GPL
-.TH FREE 1 "20 Mar 1993 " "Cohesive Systems" "Linux User's Manual"
+.TH FREE 1 "5 Oct 2009 " "Cohesive Systems" "Linux User's Manual"
 .SH NAME
 free \- Display amount of free and used memory in the system
 .SH SYNOPSIS
-.BR "free " [ "\-b" " | " "\-k" " | " "\-m" "] [" "\-o" "] [" "\-s"
-.I delay
-.RB "] [" "\-t" "] [" "\-V" ]
+.B free
+.RB [ \-b | \-k | \-m | \-g ]
+.RB [ \-c
+.IR count ]
+.RB [ \-l ]
+.RB [ \-o ]
+.RB [ \-t ]
+.RB [ \-s
+.IR delay ]
+.RB [ \-V ]
 .SH DESCRIPTION
 \fBfree\fP displays the total amount of free and used physical and swap 
 memory in the system, as well as the buffers used by the kernel.
 The shared memory column should be ignored; it is obsolete.
-.SS Options
-The \fB-b\fP switch displays the amount of memory in bytes; the 
-\fB-k\fP switch (set by default) displays it in kilobytes; the \fB-m\fP
-switch displays it in megabytes.
-.PP
-The \fB-t\fP switch displays a line containing the totals.
-.PP
-The \fB-o\fP switch disables the display of a "buffer adjusted" line.
-If the -o option is not specified, \fBfree\fP subtracts buffer memory
-from the used memory and adds it to the free memory reported.
-.PP
-The \fB-s\fP switch activates continuous polling \fIdelay\fP seconds apart. You
+.SS OPTIONS
+.TP
+\fB\-b\fR
+Display the amount of memory in bytes.
+.TP
+\fB\-c\fR \fIcount\fR
+Display the result \fIcount\fR times.  Requires the \fB\-s\fR option.
+.TP
+\fB\-g\fR
+Display the amount of memory in gigabytes.
+.TP
+\fB\-k\fR
+Display the amount of memory in kilobytes. This is the default.
+.TP
+\fB\-l\fR
+Show detailed low and high memory statistics.
+.TP
+\fB\-m\fR
+Display the amount of memory in megabytes.
+.TP
+\fB\-o\fR
+Display the output in old format, the only difference being this option
+will disable the display of the "buffer adjusted" line.
+.TP
+\fB\-s\fR
+Continuously display the result \fIdelay\fP seconds apart. You
 may actually specify any floating point number for \fIdelay\fP, 
 .BR usleep (3)
 is used for microsecond resolution delay times.
-.PP
-The \fB\-V\fP displays version information.
+.TP
+\fB\-t\fR
+Display a line showing the column totals.
+.TP
+\fB\-V\fR
+Display version information.
 .SH FILES
-.ta
-.IR /proc/meminfo "\-\- memory information"
+.nf
+/proc/meminfo memory information
 .fi
-
-.SH "SEE ALSO"
-.BR ps (1),
-.BR slabtop (1),
-.BR vmstat (8),
-.BR top(1)
-
 .SH AUTHORS
 Written by Brian Edmonds. 
 
 Send bug reports to <albert@users.sf.net>
-
+.SH "SEE ALSO"
+.BR ps "(1), " slabtop "(1), " top "(1), " vmstat (8).
+.\"{{{}}}
